modules/core/pom.xml

```diff
@@ -141,6 +141,33 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <phase>package</phase>
+            <configuration>
+              <minimizeJar>true</minimizeJar>
+              <artifactSet>
+                <includes>
+                  <include>org.apache.thrift:libthrift</include>
+                </includes>
+              </artifactSet>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <shadedClassifierName>shaded</shadedClassifierName>
```

Comment:
Do we want to modify our original jar, or have the shaded one separate? I'm thinking we probably want the shaded one to be the default artifact, for this module, rather than a separate artifact with a classifier.

Reply:
I also think it would be best to modify the original jar instead of create a separate shaded jar.

Reply:
Also, feel free to ask any quesitons I will help the best I can.  I have used the shade plugin a few times, but not recently and not for this specific case.

Reply:
@keith-turner Yes I was still working on this. Currently in Hawaii kinda off the grid for most of the day for class. Survived the "Incoming ballistic missile" false alarm though. 

I have it shaded and relocated but it is creating an uber-jar in fluo-core that has all the dependancies from the entire project and not just just that module so I want to fix that at some point.

I could probably vpn into my workstation later and push what I have changed.